src/common/config_parser.cpp

```diff
@@ -629,8 +629,12 @@ void ConfigParser::addOptionsTranslation(cli::CLIWrapper& cli) {
   addSuboptionsDevices(cli);
   addSuboptionsBatching(cli);
 
-  cli.add<bool>("--optimize",
-      "Optimize speed aggressively sacrificing memory or precision");
+  cli.add<bool>("--optimize16",
```

Comment:
Does it make sense to have both --optimize16 and --optimize8 together? Which one takes the effect then? If not, maybe use a single option `--optimize 8bit/16bit` or `--optimize intgemm8/intgemm16` with an implicit value?